### PR TITLE
more consistent terminology

### PR DESCRIPTION
**Pull request by tomerweller**
fix minor inconsistencies in terminology

### CODE REVIEW

**Comment by tomerweller** [comment]
This is just scratching the surface. But better something than nothing.

File: ecosystem/sep-0001.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ content-type: text/plain
 
 `stellar.toml` is in [TOML](https://github.com/toml-lang/toml) file format. The `stellar.toml` fields and sections are described below. You should complete as much of this as you can. Many Stellar apps, including important wallets and exchanges, make decisions on which tokens to support based on the completeness of their Account Information and Documentation sections.
 
-### Account Information
+### Organization Information
```
Comment (leighmcculloch) [inline ecosystem/sep-0001.md:45]: Nothing in this section looks like it relates to an organization to me. It's all server URLs, and accounts lists. Maybe we should call this "General Information"?

File: ecosystem/sep-0001.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ HORIZON_URL | url | Location of public-facing Horizon instance (if you offer one
 ACCOUNTS | list of `G...` strings | A list of Stellar accounts that are controlled by this domain 
 URI_REQUEST_SIGNING_KEY | Stellar public key | The signing key is used for [SEP-7](sep-0007.md) delegated signing
 
-### Issuer Documentation
+### Organization Documentation
```
Comment (leighmcculloch) [inline ecosystem/sep-0001.md:64]: 👍

File: ecosystem/sep-0001.md (hunk 1 above)
Comment (tomerweller) [reply ecosystem/sep-0001.md]: @leighmcculloch you're a modern day poet.

**Pull request merged**